src/main/java/com/enonic/app/guillotine/GuillotineConfig.java

```diff
@@ -2,5 +2,7 @@
 
 public @interface GuillotineConfig
 {
-    String queryplayground_ui_mode() default "auto";
+	String queryplayground_ui_mode() default "auto";
+
+	boolean throw_error_on_modifying_unknown_fields() default true;
```

Comment:
`too.many.dots.in.one.sentence.will.lead.to.confusion = true`

we do not split words by `.`. `.` - is namespace separator.
camelCase is what you need

`graphql.extention.modifyUnknownField = 'ignore' | 'warn' | 'throw' (default: 'throw')`

Reply:
What about just `extensions.modifyUnknownField`?